src/main/java/dino/Dino.java

```diff
@@ -0,0 +1,81 @@
+package dino;
+
+import dino.command.*;
+import dino.exception.*;
```

Comment:
Perhaps it would be better to explicitly name your imports for better documentation.